CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # Quarkus Tools for Visual Studio Code Changelog
 
+## 1.3.0 (February 6, 2020)
+
+### Enhancements
+
+ * Add new preference to disable application.properties value validation. See [#201](https://github.com/redhat-developer/vscode-quarkus/issues/201)
+ * Improve checking vscode-yaml extension. See [#195](https://github.com/redhat-developer/vscode-quarkus/issues/195)
```

Comment:
we should rather mention https://github.com/redhat-developer/vscode-quarkus/pull/189

Reply:
fixed